ibm/mas_devops/roles/ocp_updatecluster/defaults/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+# We will use the IBM Cloud CLI to update ocp cluster
+cluster_type: "{{ lookup('env', 'CLUSTER_TYPE') }}"
+ocp_fips: "{{ lookup('env', 'OCP_FIPS') | default('false', true) | bool }}"
```

Comment:
Can we make it `OCP_FIPS_ENABLED` to make it clear that we expect this to define whether it would be OCP FIPS enabled or not?

Also can we make the role name more specific than `ocp_updatecluster`? 
If it's about updating api server, maybe name it `ocp_update_apiserver` or something less generic?

Also you need to include a readme for the new role, an entry in for it `/ansible-devops/mkdocs.yml`, in `ansible-devops/build/bin/copy-role-docs.sh` and since this is a new feature, add a `[minor]` in the PR + include this in the changelogs: `ansible-devops/docs/changes.md`